client/reducers/repos.js

```diff
@@ -1,4 +1,4 @@
-import checks from './checks'
+import checks, {init as initChecks} from './checks'
```

Comment:
I don't think it's a good idea to make reducers depend on each other in that way. The hierarchy of reducer models the whole application state. Reducers can definitely be nested and respond to the same events.

I think the intention with the `checks.js` reducer was originally a different one: to have a root-level 'checks' resource for some reason (but we're not using it that way). Instead, checks are only part of the repos/repo data structure. Thus they should be a subresource.

We can create more complex reducers by nesting them. See an [example here](https://github.com/mfellner/zappr-mock/tree/master/src/reducers).